Try_Catch.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>

```

Comment:
Please create separate  files for better code structure

Reply:
Done